Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -253,11 +259,18 @@ def pull
     verify_bintray_published(bintray_published_formulae)
   end
 
+  def test_bot_user(user)
+    test_bot = ARGV.value "test-bot-user"
+    return test_bot if test_bot
+    return "BrewTestBot" if user.casecmp("homebrew").zero?
+    "#{user.capitalize}TestBot"
```

Comment:
I wonder if this is a bit magic and we should just require this argument to get passed. Thoughts?

Reply:
For a command run primarily by CI, I'd agree. Since `brew pull --bottle` is run interactively primarily, a reasonable default value is particularly helpful. How to name this test-bot user is arbitrary and may as well be consistent across taps. An alternative would be to store this value in the `.gitconfig` of each tap, or possibly in a dot file that's committed to the tap.

Reply:
I think that might an argument for accepting environment variables for some of these options instead but I'm fine as-is, too.

Reply:
They'd need to set/unset/change this environment variable depending on which tap organization they're currently working on, if they contribute to more than one tap organization. My preference is as-is.